### ISSUE TITLE
Measure isn't the same over time

### ISSUE THREAD

**Issue by yasuiniko** [jasonrwang/AB-Wildfire#5]
The amount of area destroyed by fire depends on amount of resources allocated. Also, after huge fires there's likely to be better fire preparedness in later years. The y axis should be something like number * severity of wildfires.

**Comment by jasonrwang** [comment]
By 'amount of resources allocated', do you mean the effort given to measure the extent of the fire?

Re: fire preparedness, yes, but it also takes time for things like fire breaks, lookout towers, etc. to be planned and built. Next years' prone regions will be different – the areas where a fire has already occurred aren't at risk anymore.

AB categorizes 'severity' of fire by the area that it burns, but the highest bin is 500+ acres in 1961-82, which is only 202 hectares. Here's a hist of the fire sizes (d8a4533). How do you propose we can rate 'severity'?
![loghistABWildfires_1961-2019](https://user-images.githubusercontent.com/6628281/58747499-47f3b880-846c-11e9-97a4-5f0415a95c6c.png)

**Comment by yasuiniko** [comment]
My bad, that was unclear. I came into this thinking you wanted to look more specifically at climate change. If it's more of a general data collection project the graphs aren't so important. 

Back to storytelling with graphs! I meant you should control for firefighting ability, which can be roughly estimated by some things that may or may not be recorded: number of firefighters, breakthroughs in firefighting methodology, overall firefighting budget, etc. As these resources change, I imagine Alberta's large-fire risk could change dramatically. 

If firefighting resources aren't a big influence on fire size, then it's probably not necessary to control for it. But I have a strong intuitive sense that Alberta's firefighting ability would change meaningfully over the course of 100 years.

**Comment by jasonrwang** [comment]
I see your point re: ability, but I think that's way out of scope for this. If you can propose a tidy way to account for those factors, I'd be happy to look at implementing them eventually.

Most of the provincial data (i.e. not the Canadian stuff added in #10) also records things like:
* Number of fire fighters
* Vehicles deployed
* If helicopters went in

I don't see a clear way forward to characterize the influence of those. We can't compare to a theoretical baseline (how big could the fire have gotten?), but we can perhaps see the total size of fire relative to how many fire fighting resources were deployed (should eventually correlate with smaller fires under your theory, right?). We also have more population now than 100 years ago, so do we account for that (i.e. that there are likely more people able to help fight fires)?

**Comment by yasuiniko** [comment]
You're right, it's a bit unclear what the clearest way to look at this data is. Off the top of my head normalizing per year based on regional firefighting budget or number of fire-fighters makes sense, but I'm just spitballing here. Is any of that data is available?

Are you talking about population because most fires are accidentally started by people?
